src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -107,20 +114,33 @@ public Scanner(Reader reader, ErrorHandler handler) {
      */
     public Token scan() {
         String spelling = skippedLastToken;
```

Comment:
This scan method is not elegant at all. This version is harder to read, has more unneeded LOC, and adds no functionality to main.